lib/components/viewers/pattern-row.tsx

```diff
@@ -112,7 +112,9 @@ class PatternRow extends Component<Props, State> {
               className="expansion-button"
               onClick={this._toggleExpandedView}
             >
-              <Icon type={`chevron-${this.state.expanded ? 'up' : 'down'}`} />
+              <StyledIconWrapper>
+                {this.state.expanded ? <ChevronUp /> : <ChevronDown />}
```

Comment:
Nitpick: I've been seeing this pattern a lot. I'll leave it to the second reviewer to decide if a util method that takes a boolean to display either up or down is worth it

Reply:
Putting this in a util function feels arbitrary to me, since it doesn't really reduce the complexity of anything or make it more readable.